client/src/features/sessionsV2/SessionView/SessionView.tsx

```diff
@@ -69,67 +71,70 @@ import {
 import { useGetProjectsByProjectIdMembersQuery } from "../../projectsV2/api/projectV2.enhanced-api";
 import UpdateSessionLauncherModal from "../UpdateSessionLauncherModal";
 import { ModifyResourcesLauncherModal } from "../components/SessionModals/ModifyResourcesLauncher";
-import sessionViewStyles from "./SessionView.module.scss";
 
-function SessionCard({
-  session,
-  project,
-}: {
-  session: Session;
-  project: Project;
-}) {
+interface SessionCardContentProps {
+  color: string;
+  contentDescription: React.ReactNode;
```

Comment:
I prefer:

```suggestion
  contentDescription: ReactNode;
```

It's shorter and forces to import from `react`, instead of relying on an ambient import.